lib/cli.rb

```diff
@@ -0,0 +1,148 @@
+require_relative 'mini_twitter_client'
+require 'json'
+
+class CLI
+  def initialize
+    @mini_twitter_client = MiniTwitterClient.new
+    @author = Author.new
```

Comment:
A single instance of `CLI`:

```ruby
cli = CLI.new
```

will need a single instance of `MiniTwitterClient`, but it will work with multiple authors - for example:

```ruby
cli.post_tweet
# user provides input for a single tweet with one author

cli.post_tweet
# user provides input for a second tweet with a different author
```

Currently, you would use (`@author`) the same instance of `Author` for both commands presented above. So, you would use single instance of `Author` to represent 2 different authors.

I don't think that's good. Currently it does not create other issues (it could in the future), but I guess you see that it does not fell right.

If you want to use `Author` class, create a separate instance for every author.

```diff
   def post_tweet
-      @author.name = get_input('name')
-      @author.email = get_input('email')
+      name = get_input('name')
+      email = get_input('email')
+      author = Author.new(name: name, email: email)
```

Reply:
Yes, I thought about it , but I chose that way becouse I need author only for create new tweets, but you have a right, it is not good way :)